forge-script/PermissionlessPoolWithdrawAdminFeeCheck.s.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.17;
+
+import "forge-std/Script.sol";
+import "@openzeppelin/contracts-4.4.0/token/ERC20/utils/SafeERC20.sol";
+
+interface IPermissionlessSwap {
```

Comment:
Is it possible to import existing ifaces?

Reply:
Only if the existing interface matches the solidity version :(
Since our swap contracts are using 0.6, we cant import without dependencies crashing into each other.